builtin/settingtypes.txt

```diff
@@ -642,6 +671,23 @@ bloom_radius (Bloom Radius) float 1 0.1 8
 #    Requires: shaders, enable_post_processing, enable_bloom
 enable_volumetric_lighting (Volumetric lighting) bool false
 
+[**Other Effects]
+
+#   Simulate translucency when looking at foliage in the sunlight.
+#
+#   Requires: enable_shaders, enable_dynamic_shadows
+enable_translucent_foliage (Translucent foliage) bool false
+
+#   Apply crude bump maps to nodes. It is recommended to use a tilted sun orbit to go with this (Sky Body Orbit Tilt).
+#
+#   Requires: enable_shaders, enable_dynamic_shadows
+enable_bumpmaps (Bump maps) bool false
+
+#   Apply reflections to nodes.
+#
+#   Requires: enable_shaders, enable_dynamic_shadows
+enable_node_reflections (Node reflections) bool false
```

Comment:
does this just work as one-size-fits-all effect on all nodes?
due to the assumed complexity also sounds like a good splitting target